### DISCUSSION TITLE
Is there any way to execute from path?

### DISCUSSION THREAD

**Discussion by anikghosh256** [cmderdev/cmder#2714] [Q&A] [answered]
```
λ ./vendor/bin/pint -v
'.' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by DRSDavidSoft** [comment, marked as answer]
Use Windows-style path separator (`\`) instead of *nix one (`/`) in Cmd.exe/Clink
`.\vendor\bin\pint -v`

Must also set the current working directory to CMDER_ROOT (i.e. `cd %cmder_root%`)
